settings.py

```diff
@@ -504,9 +504,8 @@ def JINJA_CONFIG():
     from django.core.cache.backends.memcached import MemcachedCache
     from django.core.cache import caches
     cache = caches['memcache']
-    config = {'extensions': ['jinja2.ext.i18n', 'tower.template.i18n',
-                             'jinja2.ext.with_', 'jinja2.ext.loopcontrols',
-                             'jinja2.ext.autoescape',
+    config = {'extensions': ['jinja2.ext.i18n', 'jinja2.ext.with_',
```

Comment:
You want to use `puente.ext.PuenteI18nExtension' and not `jinja2.ext.i18n`. Otherwise when rendering, trans blocks won't collapse whitespace and then none of your msgids will match and then THE MOON WILL TURN BLOOD AND VAMPIRE MONKEYS WILL RAIN FROM THE HEAVENS AND DESTROY THE CITIES OF THE EARTH WITH THEIR EVIL CHITTERING LAUGHTER AND MANIACAL BANANAS AND WE WILL ALL HAVE TO RUN UNDERGROUND AND USE EMACS AND OMG THE DESTRUCTION THE HUMANITY!!!! WHAT HAVE WE WROUGHT!?!?!